code/game/objects/items/devices/polycircuit.dm

```diff
@@ -18,10 +18,15 @@
 	else
```

Comment:
Github wont let me comment there, but line 16 change the check to:
if(user.get_inactive_held_item() != src)

Reply:
Fixes the picking up issue.